Doppler.Sap/Models/BusinessPartnerConfig.cs

```diff
@@ -0,0 +1,8 @@
+namespace Doppler.Sap.Models
+{
+    public class BusinessPartnerConfig
+    {
+        public string Endpoint { get; set; }
+        public string Mapper { get; set; }
```

Comment:
This property is not used, you can remove it

Reply:
Done